### PR TITLE
[IOTDB-4007]Support multiple paths in privilege control

### PR DESCRIPTION
**Pull request by yifuzhou**
## Description
demo:
image
image
All looks good.

### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: thrift-confignode/src/main/thrift/confignode.thrift [hunk 1]
```diff
@@ -174,7 +174,7 @@ struct TAuthorizerReq {
   4: required string password
   5: required string newPassword
   6: required set<i32> permissions
-  7: required string nodeName
+  7: required list<string> nodeName
```
Comment (RYH61) [inline thrift-confignode/src/main/thrift/confignode.thrift:177]:
```suggestion
  7: required list<string> nodeNameList
```

File: confignode/src/main/java/org/apache/iotdb/confignode/persistence/AuthorInfo.java [hunk 2]
```diff
@@ -145,7 +146,7 @@ public TSStatus authorNonQuery(AuthorPlan authorPlan) {
     String password = authorPlan.getPassword();
     String newPassword = authorPlan.getNewPassword();
     Set<Integer> permissions = authorPlan.getPermissions();
-    String nodeName = authorPlan.getNodeName();
+    List<String> nodeName = authorPlan.getNodeNameList();
```
Comment (RYH61) [inline confignode/src/main/java/org/apache/iotdb/confignode/persistence/AuthorInfo.java:149]:
```suggestion
    List<String> nodeNameList = authorPlan.getNodeNameList();
```

File: thrift-confignode/src/main/thrift/confignode.thrift (hunk 1 above)
Comment (yifuzhou) [reply thrift-confignode/src/main/thrift/confignode.thrift]: fix

File: confignode/src/main/java/org/apache/iotdb/confignode/persistence/AuthorInfo.java (hunk 2 above)
Comment (yifuzhou) [reply confignode/src/main/java/org/apache/iotdb/confignode/persistence/AuthorInfo.java]: fix

**Review by RYH61 — APPROVED** [review]
LGTM~

**Review by wangchao316 — APPROVED** [review]
LGTM

**Review by CRZbulabula — APPROVED** [review]
LGTM

**Pull request merged**